bblocks/import_tools/imf.py

```diff
@@ -165,16 +170,11 @@ def __load(self, indicator_list: list) -> None:
         # load data from disk
         df = pd.read_csv(f"{PATHS.imported_data}/{self.file_name}")
 
-        # filter data on indicator
-        self.data = df.loc[df["indicator"].isin(indicator_list)].reset_index(drop=True)
+        for indicator in indicator_list:
+            self.indicators[indicator] = df[df["indicator"] == indicator].reset_index(
+                    drop=True)
 
-        self.indicators = {
-            indicator: self.data.loc[self.data["indicator"] == indicator].reset_index(
-                drop=True
-            )
-            for indicator in indicator_list
-        }
-        print(f"Successfully loaded {indicator_list} SDR data")
+        self.data = df.loc[df["indicator"].isin(list(self.indicators))].reset_index(drop=True)
```

Comment:
`self.data` should store the raw data.
The filtered data is stored in the `self.indicators` dictionary.

Reply:
The above also means that the data should only be loaded if `self.data is None`, otherwise the indicator(s) should be loaded into the dict